middleware/app/Models/PlatformConnection.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * Class PlatformConnection
+ *
+ * @property int $id
+ * @property int $consumer_platform_id
+ * @property int $provider_platform_id
+ * @property string $state
+ */
+class PlatformConnection extends Model
+{
+    use HasFactory;
+
+    public function __construct(array $attributes = [])
+    {
+        parent::__construct($attributes);
+    }
+
+    /**
```

Comment:
Make sure here and in other models that you've pulled from Nokie's PR, especially enums, casts, etc.